packages/api/src/command/medical/patient/query-init.ts

```diff
@@ -53,3 +54,47 @@ export const storeQueryInit = async (cmd: QueryInitCmd): Promise<Patient> => {
     );
   });
 };
+
+export type BulkDownloadQueryInitCmd = BaseUpdateCmdWithCustomer & {
+  documentBulkDownloadProgress: Required<Pick<DocumentBulkDownloadProgress, "download">>;
+  requestId: string;
+  totalDocuments: number;
+};
+
+/**
+ * The function `storeBulkDownloadQueryInit` initalizes the `documentBulkDownloadProgress`field in a patient's data.
+ * @param {BulkDownloadQueryInitCmd} cmd - The `cmd` argument type to initialize the `documentBulkDownloadProgress` field
+ * @returns a Promise that resolves to a Patient object.
+ */
+export const storeBulkDownloadQueryInit = async (
+  cmd: BulkDownloadQueryInitCmd
+): Promise<Patient> => {
```

Comment:
Not a huge deal, it it seems better to have this and `appendDocBulkDownloadProgress` on the same file than this one here. Or just have one for each function.

Better to group by functional (bulk-download) than by non-functional (query-init) requirement.